### ISSUE TITLE
Improve Quickstarts

### ISSUE THREAD

**Issue by ravindk89** [minio/docs#448]
**Summary**

As per internal discussions, we want to produce a quickstart guide for each of our target platforms.

For this initial pass, we will create a new L1 "Quickstart" in the left-hand nav that will primarily consist of a table that links to the per-platform quickstarts.

Then each L2 is a per-platform quickstart.

The current quickstart on the Index (landing) can be simplified into accessing the Play console, downloading mc, and experimenting using the sandbox.

**Goals**

List the in-scope goals

- [ ] Linux Quickstart
- [ ] Windows Quickstart
- [ ] OSX Quickstart
- [ ] Kubernetes (generic) Quickstart
- [ ] Openshift Quickstart

**Non-Goals**

While we have existing docs on VMware, in absence of a testing environment it's not possible to put a fully vetted quickstart together. We will continue to work on getting a working environment so we can also bring the vsphere docs into parity.

**Additional context**

**Comment by ravindk89** [comment]
ToDo:

- Create new feature branch
- Create initial skeleton
- Split up work

**Comment by ravindk89** [comment]
Feature branch created.

To do work against it:

Closes #448 

# Summary

Tracking work to improve our initial quickstart experience.

# ToDo

- [ ] Linux Quickstart
- [ ] OSX Quickstart
- [ ] Windows Quickstart
- [ ] K8s Quickstart
- [ ] K8s Openshift Quickstart
- [ ] Container Quickstart

# Open Questions

- How much detail do we want to go into for per-requisites?
- Single-drive or multi-drive?

# Doing work on this branch

1. Check out the branch locally:

   ```
   git checkout -B DOCS-488 origin/DOCS-488
   ```

2. Do normal work
3. Commit the work and push to a point branch:

    ```
    git add .
    git commit -m "DOCS-448: Some sort of work done"
    git push DOCS-448-linux origin/DOCS-448-linux
    ```
4. Review and merge to feature branch

May have to iterate on the above, it's been a while.
